package.json

```diff
@@ -49,8 +49,8 @@
   },
   "homepage": "https://github.com/recharts/recharts",
   "peerDependencies": {
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "react": "^16.0.0 || ^17.0.0",
+    "react-dom": "^16.0.0 || ^17.0.0"
```

Comment:
Should it be this?

```js
    "react": "^17.0.0",
    "react-dom": "^17.0.0"
    ```

Reply:
Should the package version also be bumped?

```js
  "name": "recharts",
  "version": "2.0.1",
  ...
```

Reply:
As you wish.
- Allowing 16 & 17 means that the package will accept both instead of forcing only one. React 16 and 17 are compatible en theory so I feel you shouldn't bother playing with maintaining 2 versions of recharts
- the version needs a bump but major ? Minor ?

Reply:
I think you’re right, ignore my 16/17 comment. Thanks!

Reply:
Worth noting that some dependencies of recharts need the same annoying patch for the installation to work without warnings.